distribution/kubernetes/vector-all/resources.yaml

```diff
@@ -117,7 +117,7 @@ spec:
         - name: vector
           securityContext:
             {}
-          image: "timberio/vector:latest-alpine"
```

Comment:
I'm curious why the switch from alpine to debian?

Reply:
`alpine` uses `musl`, and musl has some perf issues. `debian` uses `glibc`.
`debian` shown about 20%  higher throughput than `alpine` in our load tests.